add: Turkey folder created.

ramazansancar opened the pull request:
add: İstanbul & İzmir & Ankara & Balıkesir & Marmaris cams added.
upd: Fetching status console message added.
upd: Camel case & lint rules refactored.
upd: console.log to console.info updated.
add: local control for http protocol alert added.
add: Embed stream support added.
upd: Readme for markdownlint rules updated.
add: pyScrapper added for Python Language data scrapper added. Thanks @keyiflerolsun 

Live/Preview: https://ramazansancar.github.io/AidanWelch_OpenTrafficCamMap/map.html

Added Cam List (18 Location):

- [ ] Antalya, Türkiye
  - [X] Alanya, Antalya, Türkiye
- [X] Balıkesir, Türkiye
  - [X] Sındırgı, Balıkesir, Türkiye
  -  [X] Bandırma, Balıkesir, Türkiye
- [ ] Bursa, Türkiye
  - [X] Karacabey, Bursa, Türkiye
- [X] Denizli, Türkiye
- [X] Erzurum, Türkiye
- [X] Giresun, Türkiye
- [X] İstanbul, Türkiye
- [X] İzmir, Türkiye
- [X] Kahramanmaraş, Türkiye
- [X] Konya, Türkiye
- [ ] Kocaeli, Türkiye
  - [x] Çayırova, Kocaeli, Türkiye 
- [X] Kütahya, Türkiye
- [ ] Muğla, Türkiye
  - [X] Marmaris, Muğla, Türkiye
- [X] Rize, Türkiye
- [X] Trabzon, Türkiye
- [ ] Zonguldak, Türkiye
  - [X] Çaycuma, Zonguldak, Türkiye

ramazansancar committed:
add: Turkey folder created.
add: İstanbul & İzmir & Ankara cams added.
upd: Fetching status console message added.
upd: Camel case & lint rules refactored.
upd: console.log to console.info updated.
add: local control for http protocol alert added.
add: Embed stream support added.
upd: Readme for markdownlint rules updated.

AidanWelch commented:
Checking the code now, also funny because I'm in Istanbul right now!

ramazansancar commented:
> Checking the code now, also funny because I'm in Istanbul right now!

Enjoy!

AidanWelch commented:
@ramazansancar Sorry for the late response, I was busy with some work.  I appreciate this PR, but I can't accept the Ankara script because it references your private site to scrape the data.   I am looking to only include government/otherwise official sources.

ramazansancar commented:
@AidanWelch When I have time I will remove that area and update it.

AidanWelch commented:
Thank you!

ramazansancar committed:
del: ankara deleted.

ramazansancar committed:
upd: update data

ramazansancar committed:
fix: resolve conflict.

ramazansancar committed:
fix: resolve conflict

ramazansancar committed:
fix: resolve conf

ramazansancar commented:
Must be operational now @AidanWelch Check pls

ramazansancar commented:
@AidanWelch Check pls

ramazansancar commented:
@AidanWelch Check pls

keyiflerolsun committed:
Balıkesir Belediyesi Test

keyiflerolsun committed:
Balıkesir Belediyesi Test

keyiflerolsun committed:
Balıkesir Belediyesi Test

keyiflerolsun committed:
Balıkesir Belediyesi Test

keyiflerolsun committed:
Marmaris Belediyesi Test

keyiflerolsun committed:
🕊

🕊

keyiflerolsun committed:
🕊

🕊

keyiflerolsun committed:
🕊

🕊

ramazansancar committed:
add: Balıkesir added.
upg: upgrade dependencies.
update: data updated.

ramazansancar committed:
Merge branch 'master' into master

ramazansancar committed:
Merge pull request #1 from keyiflerolsun/master

Balıkesir & Marmaris Belediyesi

Thanks @keyiflerolsun

ramazansancar committed:
add: istanbul replaceAll aded.
fix: Json conflict "," error fixed.

keyiflerolsun committed:
🕊

🕊

keyiflerolsun committed:
🕊

🕊

ramazansancar committed:
add: marmaris added.
fix: balikesir lat, long format fixed.
update: update datas.

ramazansancar committed:
upd: update data
fix: balikesir empty location fixed.

ramazansancar committed:
Merge branch 'master' into master

ramazansancar committed:
Merge pull request #3 from keyiflerolsun/master

pyScrapers Fix

keyiflerolsun committed:
added: Alanya

keyiflerolsun committed:
added: Trabzon

keyiflerolsun committed:
added: Rize

keyiflerolsun committed:
added: Tekirdağ

keyiflerolsun committed:
🕊

🕊

keyiflerolsun committed:
🕊

🕊

ramazansancar committed:
Merge pull request #4 from keyiflerolsun/master

added: alanya, trabzon, rize, giresun, erzurum, caycuma

AidanWelch committed:
Merge branch 'master' of https://github.com/AidanWelch/OpenTrafficCam into pr/ramazansancar/29

keyiflerolsun committed:
fix: pyScrapers workflow

keyiflerolsun committed:
pyScrapers

ramazansancar committed:
Merge pull request #5 from keyiflerolsun/master

fix: pyScrapers workflow

AidanWelch commented:
Hi, @ramazansancar just a question about this, where did the county names come from?

keyiflerolsun committed:
pyScrapers

ramazansancar commented:
HI, @AidanWelch Since there is no state system in our country, provinces can be divided into districts. You can think of it like towns in Europe.

keyiflerolsun committed:
pyScrapers

keyiflerolsun committed:
pyScrapers

keyiflerolsun committed:
pyScrapers

keyiflerolsun committed:
pyScrapers

keyiflerolsun committed:
pyScrapers

keyiflerolsun committed:
pyScrapers

keyiflerolsun committed:
pyScrapers

keyiflerolsun committed:
pyScrapers

ramazansancar commented:
Is there something you're waiting for to merge the PR? @AidanWelch

keyiflerolsun committed:
pyScrapers

keyiflerolsun committed:
pyScrapers

keyiflerolsun committed:
pyScrapers

ramazansancar commented:
Then I'll wait for news. @AidanWelch

ramazansancar committed:
fix: workflow fixed.

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

keyiflerolsun committed:
pyScrapers.helpers fix

ramazansancar committed:
Merge pull request #6 from keyiflerolsun/master

pyScrapers.helpers fix

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

ramazansancar committed:
fix: maps api fixed.

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

ramazansancar committed:
upd: Turkey data updated.
upd: Python scraper updated.

ramazansancar committed:
upd: map name to lat long api changed.

github-actions[bot] committed:
pyScrapers

ramazansancar committed:
upd: cam lat long finder name fixed.

github-actions[bot] committed:
pyScrapers

ramazansancar committed:
fix: location api parameters fixed.

github-actions[bot] committed:
pyScrapers

ramazansancar committed:
fix: workflow autotest only main repo running.

github-actions[bot] committed:
pyScrapers

ramazansancar committed:
add: konya added.
chore: sort providers.
fix: Fixed incorrect location pinning.

ramazansancar committed:
add: PyScrapper Readme created.
upd: requirements.txt updated.

ramazansancar committed:
add: Denizli added.
add: Global States added in map.html
fix: Locations updated.
upd: Province and district distinctions were made.
add: A cache system was created to revise location requests.
add: Added `nocache` parameter to the file extension to ensure that Json files are always up to date.

ramazansancar committed:
add: Kahramanmaraş added.

ramazansancar committed:
fix: dependencies fixed.

ramazansancar committed:
fix: dependencies list fixed.

ramazansancar committed:
fix: workflows bug fixed.

ramazansancar committed:
fix: syntax error fixed.
upd: cache cleared. Updated data.

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

ramazansancar committed:
upd: Cache updated.
fix: Workflows fixed.

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

ramazansancar commented:
Are you around here? @AidanWelch

AidanWelch commented:
@ramazansancar Sadly limited in time and by extension monetarily in how much I can work on this project 🥲.  I haven't forgotten, just unclear when I'll have a chance to properly review this

ramazansancar commented:
@AidanWelch It won't actually be a very long review process. The longer it goes on, the more code will need to be reviewed. This will happen with the addition of new locations.

AidanWelch commented:
@ramazansancar It would be a bit extensive though because there are various kinda arbitrary requirements I'm enforcing to try to ensure some semblance of coherence in this project.  For example: not reforming the existing code until it is updated, translating the python scripts into JS, and I would have to independently investigate the camera sources and ensure they're official.

ramazansancar commented:
@AidanWelch Sure, thanks

ramazansancar committed:
add: Sındırgı, Balıkesir, Türkiye added.
upd: data updated.
fix: try cactch added.

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

ramazansancar committed:
add: Kütahya added.

ramazansancar committed:
add: Karacabey/Bursa added.

ramazansancar committed:
add: Çayırova/Kocaeli added.

ramazansancar committed:
add: Çayırova/Kocaeli data added.

ramazansancar committed:
upd: Erzurum new camera provider updated.

ramazansancar committed:
add: Bandırma/Balıkesir added.

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

ramazansancar commented:
Hi @AidanWelch

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers

github-actions[bot] committed:
pyScrapers
